src/ctl/ctl.c

```diff
@@ -416,11 +402,14 @@ int ctl_query(struct ctl *ctl, void *ctx, umf_ctl_query_source_t source,
         errno = EINVAL;
         goto out;
     }
+    const char *extra_name = &name[0];
+    if (strstr(extra_name, "by_handle") != NULL) {
+        extra_name = &name[0] + name_offset;
```

Comment:
Please do not include UMF specyfic soultions in ctl* files.

Reply:
Done